docs/install/quickstart-install.md

```diff
@@ -1,6 +1,7 @@
+--8<-- "quickstart-prereqs.md"
 --8<-- "quickstart-install.md"
```

Comment:
Think we also should add the install-kn.md snippet here  (the prereqs mention installing `kn` and reference the details will be in the next section...)